src/__tests__/helpers.ts

```diff
@@ -345,6 +359,18 @@ export const ContentPatch = {
         ofType: null,
       },
     },
+    {
+      name: 'myFulltext',
```

Comment:
does a fulltext attribute really make sense as patch input?

Reply:
do you mean the naming?